documentation-website/cypress/e2e/imprint.cy.ts

```diff
@@ -11,7 +11,7 @@ describe('imprint', () => {
     cy.url().should('eq', BASE_URL + page.imprint,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Imprint | @idrinth/api-bench',);
+    cy.title().should('eq', 'Imprint | @idrinth-api-bench/api-bench',);
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted